AIPs/aip-5.md

```diff
@@ -0,0 +1,207 @@
+
+**_Alternating Series Example_**
+
+Consider a price series that alternates between $0.5 and $2, every 24hrs, infinitely:
+
+<p align="center">
+<img src="../assets/aip-5/series.png" alt="drawing" width="320"/>
+</p>
+
+For fixed-supply assets, the `market_cap` in our example above simply alternates between two values.  However, for any `rebase_reaction_lag` value other than 1, the current Ampleforth supply policy will drift upwards over time. 
+
+### Motivation for Capped Expansion Rate
+
+We also propose asymptotic limits on the rates of supply change, in order to reduce the protocol's sensitivity to short-lived, but extreme market conditions.
+
+## Specification
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+<!--This is a high level overview of *how* the AIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+The smart contract upgrade replaces the current linear rebasing function with a "balanced" sigmoid-shaped curve that horizontally asymptotes away from the origin. 
+
+### Rationale
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+Below we'll quickly review the basic sigmoid, and then explain the rationale for the "balanced" sigmoid proposed.
+
+#### 1. Sigmoid
+A sigmoid function takes the following shape, notice the horizontal asymptotes and the maximum slope at the target price($1):
+
+<p align="center">
+<img src="../assets/aip-5/basic_sigmoid_latex_thin.jpg" width="700"/>
+</p>
+
+**1.1. _Sigmoid Function and Parameters_**
+
+This equation takes, `x`, the normalized difference between `VWAP` and the price target. It returns, `Y`, the corresponding supply change percentage.
+
+```
+Y = supply change %
+x = normalized price deviation
+```
+
+<p align="center">
+<img src="../assets/aip-5/basic_sigmoid_eq_2.png" alt="drawing" width="380"/>
+</p>
+
+It has shaping parameters that determine: lower asymptote, upper asymptote, and the steepness of the curve (ie: growth rate) around its origin.
+
+```
+L = lower asymptote
+U = upper asymptote
+B = growth rate
+```
+
+#### 2. "Balanced" Sigmoid
+
+Although the Sigmoid is a good start, we can improve upon it by scaling supply changes such that they “balance” one another. More specifically:
```

Comment:
The modification applied here is not a scale modifier but the shape for the contraction side is constructed differently. 
```suggestion
Although the Sigmoid is a good start, we can improve upon it by modifying the supply changes such that they “balance” one another. More specifically:
```

Reply:
can we say transforming?